modules/cnv-editing-kubevirtstorageclassdefaults-cli.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_virtual_machines/cnv_virtual_disks/cnv-storage-defaults-for-datavolumes.adoc
+
+[id="cnv-editing-kubevirtstorageclassdefaults-cli_{context}"]
+= Editing the `kubevirt-storage-class-defaults` ConfigMap in the CLI
+
+You can edit the default storage setting by modifying the `kubevirt-storage-class-defaults` ConfigMap in the `openshift-cnv` namespace.
+
+If you want to create DataVolumes using different storage settings, you can add storage settings for storage classes in the ConfigMap.
+It is important that you configure storage settings that are supported by the underlying storage.
+
+.Procedure
+
+. Use `oc edit` to edit the Config Map:
++
+----
+$ oc edit configmap kubevirt-storage-class-defaults -n openshift-cnv
+----
+
+. Update the `data` values of the Config Map:
++
+[source,yaml]
+----
+...
+data:
+  accessMode: ReadWriteOnce <1>
+  volumeMode: Filesystem <2>
+  <new>-sc.accessMode: ReadWriteMany <3>
```

Comment:
Should be `<new>.accessMode: ReadWriteMany` because there is not expected to be a -sc suffix.